kubernetes/admission/example_loadbalancer.rego

```diff
@@ -10,7 +10,7 @@ deny[explanation] {
 	loadbalancer.is_external_lb
 	namespace := input.request.namespace
 	name := input.request.object.metadata.name
-	not whitelisted[{"namespace": namespace, "name": name}]
+	not whitelisted[{"name": name, "namespace": namespace}]
```

Comment:
This doesn't need to be changed.

Reply:
@timothyhinrichs I couldn't get `make check` to pass without it.

Reply:
Weird.

Reply:
I've added the change back to fix CI for the project. Not sure what changed.